reconstruction/rtpc/src/main/java/org/jlab/rec/rtpc/hit/TrackDisentangler.java

```diff
@@ -22,85 +26,181 @@ public class TrackDisentangler {
     private double maxdeltaphi = 0.10;
     private double maxdeltazgap = 10;
     private double maxdeltaphigap = 0.12;
-    
-    public TrackDisentangler(HitParameters params, boolean disentangle){       
+    private double c0 = 20;
+    private double c1 = 0.07;
+    private double c2 = 8;
+    private double c3 = 240;
+    private int eventnum = 777;
```

Comment:
it seems this eventnum and the corresponding variable in the constructor are leftover from some debugging, right? can they be removed?

Reply:
c0, c1, c2, and c3 must be defined here. I removed the initialization for them as they have to be reading the constants from ccdb. Regarding,      private int eventnum = 777; 
It has been removed as it got deined in the RTPC engine earlier.